combozork.js

```diff
@@ -1,254 +0,0 @@
-const readlineInterface = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-    return new Promise((resolve, reject) => {
-        readlineInterface.question(questionText, resolve);
-    });
-}
-let status = 5 // set for sobriety
-let inventory = ["cellphone", "lighter", "flashlight"]; //starting inventory
-function displayInventory() {
-    inventory.forEach(article => console.log(article)) // to diplay items in inventory array
-}
-class Item {
-    constructor(name, takeable, useable, content) { // possibilities for each item in inventory
-        this.name = name
-        this.takeable = takeable
-        this.useable = useable
-        this.content = content
-    }
-
-    take() {
-        if (this.takeable) {
-            inventory.push(this.name); // when used puts takeable items into inventory
-            console.log("You picked up " + this.name)
-        } else {
-            console.log("This cannot be moved!")
-        }
-    }
-    use() { // if useable lets articles in inventory be used
-        if (this.useable) {
-            if (inventory.includes(this.name)) {
-                return "used"
-            }
-            else {
-                console.log("you do not have this item in your inventory")
-            }
-        } else {
-            return "You cannot use this item now"
-        }
-    }
-    drop() {
-        inventory.filter(article => this.name !== article)
-        console.log("You have dropped the " + this.name)
-
-
-    }
-
-} // all items
-let key = new Item("key", true, true, "you now have a key")
-let flashlight = new Item("flashlight", true, true, "The flashlight lights up your surroundings ")
-let cellphone = new Item("cellphone", true, true, "You take your phone out of your pocket. \nYou have no signal but your flashlight works")
-let note = new Item("note", false, false, 'The note says "Come on in! The code for the door is 8675309"')
-let beer = new Item("beer", true, true, "You crack open the ice cold beer")
-let truly = new Item("truly", true, true, "you know better. Put the truly down \nimmediately before someone sees you")
-console.log("You’re on your way to a party in the middle of nowhere. \nYou entered the address into maps but \nyou lose cell service after you make your \n10th turn on some desolate dirt road.  \nTen minutes later you’re about to turn around \nwhen the dashboard lights up like a Christmas tree. \nYour car sputters and dies. \nYou turn the key again and again but no dice. \nYou flick on your high beams to try to \nsee if anything is up ahead. \nYou think you can make out the silhouette \nof barn further down the road. \nIt’s the first sign of civilization you’ve \nseen in the last 15 minutes. \nIt’s a very long walk back to the last \nhouse you passed, its pitch black and \nits starting to get cold. ")
-console.log("You can go to the barn or walk back the way you came")
-const opening = "What would you like to do? \n>_"
-play();
```

Comment:
Great use of class constructor, you may want to consider replicating this for your rooms if you decide to refactor